.github/workflows/helm-template-check.yml

```diff
@@ -0,0 +1,87 @@
+name: Helm Template Check
+
+on:
+  pull_request:
+    paths:
```

Comment:
We should also run when loculus commit pointed at changes

Reply:
that's in pathoplexus_app?